### DISCUSSION TITLE
How to export mac lazer to windows?

### DISCUSSION THREAD

**Discussion by gabrieldabbah** [ppy/osu#23299] [Q&A] [answered]
I bought a new windows PC. I have lazer installed on my Mac. I want to export all collections, maps, etc. to my windows pc. How can I do that? Thank you for the support.

**Comment by Joehuu** [comment, marked as answer]
I would do the following (on both computers):
- open/install osu!
- press the `Open osu! folder` button in settings (to easily navigate the `osu` folder)
- close osu!
- go up a directory

Then, I would replace the `osu` folder on the Windows pc with the Mac one (copying via external drive, etc) in the same place.
